### PR TITLE
Add MongoDB help page

### PR DESCRIPTION
**Pull request by TDaglis**

### CODE REVIEW

File: _databases/mongodb.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+### Database information:
+Version: 3.0  
+**Host**: `mongo`  
+
+You can connect to your database from your SourceLair Terminal using:  
+`mongo --host mongo`
+
+
+**Note**: If you connect a new database to an existing project, you will have to reload your Terminal before being able to access it.
```
Comment (akalipetis) [inline _databases/mongodb.md]: This is no longer valid, since the terminal is forced to restart alongside the rest of the user's processes.

Comment (akalipetis) [inline _databases/mongodb.md]: Could you also by the way change it [here](https://github.com/sourcelair/help/pull/17/files#diff-0635d627a0201fc29142c98a45032aa5R24) too?

**Comment by akalipetis** [comment]
Minor comment, apart from it LGTM :bear:

**Comment by parisk** [comment]
Apart from @akalipetis' comment, LGTM :cake:.

**Comment by TDaglis** [comment]
Resolved comments

**Comment by akalipetis** [comment]
LGTM :bear:

**Pull request merged**